amf-aml/shared/src/test/scala/amf/testing/parsing/DialectsParsingTest.scala

```diff
@@ -434,6 +434,18 @@ trait DialectsParsingTest extends DialectTests {
         directory = s"$basePath/conditional"
     )
   }
+
```

Comment:
Why comment this ?

Reply:
The test is necessary but is not working. This is what I told you about IRI resolution in JSON-LD parsing.